README.md

```diff
@@ -23,11 +23,17 @@ This example app uses Node, Express, Webpack, React, Redux, and Shopify/polaris
 
 ## Running the project locally
 
-### Run easily with Docker without install dependencies
-
-1. Define `SHOPIFY_APP_SECRET` as environment variable with the right value.
-2. Run `./run-with-docker.sh`
-3. Open <http://localhost:3000>
+### Build and run tests with Docker (without install dependencies)
+
+1. Run `./scripts/run-with-docker.sh`, in windows `sh ./scripts/run-with-docker.sh` 
+### Build and run tests locally
```

Comment:
```suggestion

### Build and run tests locally
```

Reply:
In markdown, it is necessary using interlines to split paragraphs. Also, before and after a tittle an interline is required.

In lists, the interline between items is not necessary, but it is also accepted and in some cases is very useful.

There is a nice VS Code plugin that could help us top learn markdown: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
Very nice, thanks!